terragrunt/aws/load_balancer/security_groups.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_security_group" "cra_upd_loadbalancer_sg" {
+  name        = "cra-upd-loadbalancer-sg"
+  description = "LoadBalancer Security Group for CRA UPD"
+  vpc_id      = var.vpc_id
+}
+
+resource "aws_security_group_rule" "cra_upd_sg_rule_ingress_vpc_to_lb" {
+  type              = "ingress"
+  description       = "Allow ingress from VPC on port 80"
+  from_port         = 80
```

Comment:
I would ingest traffic form https, therefore port 443. You should always make it that your site uses https instead of http.

Reply:
If traffic coming into the website is handled using https using the api gateway, that is fine. As long as HTTPS outside connections are forced, that is fine.